typedb/connection/cluster/client.py

```diff
@@ -41,20 +40,17 @@ class _ClusterClient(TypeDBClusterClient):
     def __init__(self, addresses: Iterable[str], credential: TypeDBCredential, parallelisation: int = None):
         self._credential = credential
         self._server_clients: Dict[str, _ClusterServerClient] = {addr: _ClusterServerClient(addr, credential, parallelisation) for addr in self._fetch_server_addresses(addresses)}
-        self._stubs = {addr: client.connection_factory().newTypeDBStub(client.channel()) for (addr, client) in self._server_clients.items()}
         self._database_managers = _ClusterDatabaseManager(self)
         self._cluster_databases: Dict[str, _ClusterDatabase] = {}
         self._user_manager = _ClusterUserManager(self)
         self._is_open = True
-        print("Cluster client created")
 
     def _fetch_server_addresses(self, addresses: Iterable[str]) -> Set[str]:
         for address in addresses:
             try:
                 print("Fetching list of cluster servers from %s..." % address)
                 with _ClusterServerClient(address, self._credential) as client:
-                    typedb_cluster_stub = client.connection_factory().newTypeDBStub(client.channel())
-                    res = typedb_cluster_stub.servers_all(cluster_server_manager_all_req())
+                    res = client.stub().servers_all(cluster_server_manager_all_req())
```

Comment:
Reusing the same `stub()` from the `client` object, rather than creating a new one.